qiskit/pulse/library/symbolic_pulses.py

```diff
@@ -1023,3 +1023,133 @@ def __new__(
         instance.validate_parameters()
 
         return instance
+
+
+def GaussianRiseEdge(
```

Comment:
I think `GaussianSquareRise` and `GaussianSquareFall` might be clearer names since they do potentially include a little bit of the flat top of the pulse.